dist/index.d.ts

```diff
@@ -782,8 +744,6 @@ interface GPUPrimitiveState {
   frontFace?: GPUFrontFace;
   cullMode?: GPUCullMode;
   unclippedDepth?: boolean;
-  /** @deprecated */
-  clampDepth?: boolean;
```

Comment:
I don't think Chrome fixed this yet so we may need to do some `any` casting in that test for now. (That's fine though.)

Reply:
Put it back in for the time being, we can always get it on the next pass.